README.md

```diff
@@ -6,22 +6,44 @@ Building blocks for eye tracking in Augmented Reality `AR` and Virtual Reality `
-4. (Windows)Set the Pupil Service App Path to the path where your pupil_service.exe is stored by clicking on the Browse button.
-5. _[HMD Hardware Dependent]_ Some headsets like the HTC Vive require the Open VR SDK. This SDK is is not enabled in the project by default. You can add it by going to `Edit > Project Settings > Player > Other Settings` and selecting it using the `+` symbol. Make sure that `Virtual Reality Supported` is also checked
-6. Plugin your HMD and start it up.
-7. Press the `Play` button in the Unity Editor. (This starts Pupil Eye Processes; check to make sure that the pupil is well detected).
-8. Choose the calibration method under 'PupilGaze' GameObject >> Calibration >>2D/3D
-9. To calibrate press the `C` key, or click on the Calibrate button
-
-## Develeoper Setup
-
-1. Open `Assets/scene/Calibration.unity` with the Unity3d Editor.
-2. Select the GameObject called 'PupilGaze'
-3. Under Settings, tick the ShowAll checkbox, this will enable you to see all exposed variables.
-4. There are 5 Syles that can be adjusted. Main Tabs, Settings Labels, Settings Values, Settings Browse, Logo.
-5. The Default paths and file names for the Pupil Service App are stored under the Platforms variable for each platform. You can add one manually if your current platform is not listed.
-6. In the current version the 2D and 3D calibration points are not exposed in the final GUI, but you can find and edit them here until the interface comes out. Relevant variables : Calib Points 2D, Calib Points 3D.
+3. Select the PupilGaze GameObject in Unity Hierarchy.
+
+4. Connection
+	* #### Local
+		1. In Unity's Inspector select Settings>>Pupil App>>Local
+		2. Click on the Browse button and navigate then select the pupil_service.exe/pupil_capture.exe you want to use.
+		3. Make sure that your Service Port under the path matches the one you have in your Pupil Capture!
+	* #### Remote
+		1. In Unity's Inspector select Settings>>Pupil App>>Remote
+		2. Make sure that your Pupil Capture application is running on the remote machine.
+		3. Take note of the IP address of the active communication device in your remote machine. (Please keep in mind that in some cases the IP address stated in Pupil Capture may not be correct, check the IP address of your system!).
+		4. Copy this IP address to the IP address field in Unity.
+		
+5. Still under Settings>>Pupil App you have the option to toggle Autorun Pupil Service when Play is pressed in Unity, by clicking on the Autorun Pupil Service button. Please note that once Autorun in turned off in Unity's Play mode you will have the option to Start/Stop Pupil Service. These options are right under the above mentioned toggle button.
+6. Establish connection . Make sure that the line under the Pupil Labs logo says (connected) and both eye Icons are green( these are indicating that each eye process has started successfully thus both eye camera windows are running in the Pupil App ).
```

Comment:
Edit:

Confirm connection - Make sure that the plugin status (below the Pupil Labs logotype) displays the word "connected" and that both of the eye icons are green. The green eye icons signify that the eye processes are running.